package.json

```diff
@@ -8,6 +8,7 @@
     "email": "pablo.molina@liferay.com"
   },
   "files": [
+    "bundler-config.js",
```

Comment:
As suggested in Slack private conversation, I would move this to `src` and export it as a proper API referenced by the `main` entry of the `package.json`